webapp/apps/taxbrain/views.py

```diff
@@ -602,9 +632,49 @@ def submit_micro(request, pk):
     model.tax_result = None
 
```

Comment:
The data is already parsed into a `TaxSaveInputs` object so that means that there is no need to go through the `request` parsing logic in `submit_reform`.